expressEngine/product/service/product.js

```diff
@@ -25,6 +25,26 @@ class Service {
     }
     return { data: product, message: success.productRegistered };
   }
+
+  getProduct = async ({ query }) => {
+    const dataToFind = {
+      collection: Product,
+      data: {
+        isDiscontinued: { $exists: false },
```

Comment:
@aman-successive it is still wrong. Set check as isDiscontinued is not true

Reply:
updated it now